.github/workflows/docker.yml

```diff
@@ -0,0 +1,53 @@
+name: "Build docker image"
```

Comment:
Please check if it works (you may modify the workflow to trigger the image build from this branch)

Reply:
checked